lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,29 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = StringUtils.interpolate('<a href="{url}">Account Settings page.</a>',
```

Comment:
I agree that doesn't look nice.  However, I also don't understand what didn't work about it.

I'm not sure if you think the following looks better.  However, ultimately, you may run into whatever other problem you were already running into.
```
var accountSettingsLink = HtmlUtils.interpolateHtml(
   HtmlUtils.HTML('<a href="{url}">Account Settings page.</a>'),
   ...
```